src/components/ordersPage/content.js

```diff
@@ -4,21 +4,142 @@
+        let cont_orders = $('<div  id="orders-container"></div>');
+        let cont_del_orders = $('<div id="delivered-orders-container" ></div>');
+
+        $(all_orders).append(orders);
+        $(all_orders).append(del_orders);
+        $(del_orders).append(header_del);
+        $(orders).append(header_order);
+        $(del_orders).append(cont_del_orders);
+        $(orders).append(cont_orders);
+
+        for (let i = 0; i < this.orders.length; i++){
+            $(cont_orders).append(this.createOrderCard( this.orders[i], i));
+        }
+         $(cont_orders).on("drop", (ev) => {
+            ev.preventDefault();
+            var data = ev.originalEvent.dataTransfer.getData("text");
+            ev.target.appendChild(document.getElementById(data));
+             ev.stopPropagation();
+        });
+
+        $(cont_orders).on("dragover", (ev) => {
+           ev.preventDefault();
+           ev.stopPropagation();
+        });
+
+        $(cont_del_orders).on("drop", (ev) => {
+            ev.preventDefault();
+            var data = ev.originalEvent.dataTransfer.getData("text");
+            ev.target.appendChild(document.getElementById(data));
+            ev.stopPropagation();
+        });
+
+        $(cont_del_orders).on("dragover", (ev) => {
+           ev.preventDefault();
+           ev.stopPropagation();
+        });
+
+        return all_orders;
+    }
+    
+    
+    createOrderCard = (order_cont,id) => {
+        let cont_card = $('<div draggable="true" id="' + id + '" class="container-card"></div>');
+        let card = $('<div class="order-card"></div>');
+        let content_card = $('<div class="content-card"></div>');
+
+
+        $(cont_card).on("dragstart", (ev) => {
+            console.log(ev.originalEvent.dataTransfer);
+            ev.originalEvent.dataTransfer.setData("text", ev.target.id);
+            ev.stopPropagation();
+        });
+
+
+
+        let table = $('<div class="container-table">'+
+                        '<div class="txt-table"><span data-textid="order-table"></span>:</div>'+
+                        '<div class="table"> ' + order_cont.table + '</div>'+
+                    '</div>');
+
+        let time = $('<div class="container-time">'+
+                        '<div class="txt-time"><span data-textid="order-time"></span>:</div>'+
+                        '<div class="time"> ' + order_cont.time + '</div>'+
+                    '</div>');
+
+        let order = $('<div class="container-order">'+'<div class="txt-order"><span data-textid="order-order"></span>:</div>')
+
+
+        for (let i = 0; i < order_cont.items.length; i++){
+            $(order).append(this.addItems( order_cont.items[i]));
+        }
+
+
+        $(cont_card).append(card);
+        $(card).append(content_card);
+        $(content_card).append(table);
+        $(content_card).append(time);
+        $(content_card).append(order);
+     
+
+       return cont_card;
```

Comment:
Make it a more read-friendly order (and it makes more sense in the suggested order, since you return cont_card you'd like to the thing to append to cont_card last)
```suggestion
        $(content_card).append(table);
        $(content_card).append(time);
        $(content_card).append(order);

        $(card).append(content_card);

        $(cont_card).append(card);
        
        return cont_card;
```